### DISCUSSION TITLE
Extended arm

### DISCUSSION THREAD

**Discussion by Bwachtel** [Thomas-3D/Front_Decontaminator#1] [Ideas]
So I have a 350mm 2.4 and my bed has built in sheet stops that interferes with all of the nozzle scrubbers. I also have kinematic mounts. Is there any way you could have a version of the brush that extends to the corner of the bed but have the extension part stay below the bed?

**Comment by Thomas-3D** [comment]
Hi, can you tell me the distance between the acryl sheet and your underside of the bed? I don't have kinematic mounts

**Reply by Bwachtel** [reply]
Will do first think when I get home!

This is the bed btw.

https://preciseprinterparts.com/voron-cast-aluminum-printer-bed-350mm-vor-ver-24-and-18.html

**Comment by Bwachtel** [comment]
To top of bed with magnetic sheet and flex plate is 45.58mm
Bed thickness plus magnetic sheet and flex plate is 9.9mm

**Comment by Bwachtel** [comment]
distance from outer extrusion edge to edge of bed corner is ~95mm
distance from outer edge of sheet stop to edge of bed corner is ~64mm

**Comment by Bwachtel** [comment]
So was this something you think you can do?
